modules/cmsis-dsp/CMakeLists.txt

```
@@ -19,6 +19,8 @@ if(CONFIG_CMSIS_DSP)
     ${cmsis_glue_path}/CMSIS/Core/Include
   )
 
+  zephyr_library_compile_definitions(ZEPHYR_USE_TOOLCHAIN_STDINT)
```

Do we really need a new hidden define for this ? (`ZEPHYR_USE_TOOLCHAIN_STDINT`).

Why not just use:
```suggestion
  /* Define ZEPHYR_INCLUDE_TOOLCHAIN_STDINT_H_ to avoid using Zephyr's stdint. */
  zephyr_library_compile_definitions(ZEPHYR_INCLUDE_TOOLCHAIN_STDINT_H_)
```

That will effectively achive the same.

---

yeah, I actually did that first and thought that maybe having an explicit symbol for this would be less confusing? Perhaps not? I'll switch it back.